crates/shared/src/order_quoting.rs

```diff
@@ -883,7 +896,11 @@ mod tests {
                 from: H160([3; 20]),
                 ..Default::default()
             }),
-            signing_scheme: QuoteSigningScheme::Eip712,
+            signing_scheme: QuoteSigningScheme::Eip1271 {
+                onchain_order: false,
+                verification_gas_limit: 1,
+            },
+            additional_gas: 2,
```

Comment:
The quote’s `gas` amount was `3`, and now we have an additional `1 + 2` gas from signature verification...